lib/actions/sayall.js

```diff
@@ -121,7 +122,16 @@ function sayAll(player, values) {
     }
   };
 
-  return tryDownloadTTS(text, language)
+  let fileURI;
+
+  if text.startsWith("static_tts_") {
```

Comment:
This whole `static_tts` prefix thing is a total hack. I'm open to either better ideas and/or if you think it should be a standalone API like `/living room/static_file/prerecorded_file_key` or something.